web-client/src/styles/custom.scss

```diff
@@ -2355,7 +2355,15 @@ button.change-scanner-button {
   padding: 0;
 }
 
+.trial-session-public-card {
+  line-height: 24px;
+}
+
+.h1-size {
+  font-size: 2em;
+}
```

Comment:
Our current use of h1 on some trial session views violates axe. This is a silly class so that we can use an h2 that looks the same as before when it was an h1.